content/es/docs/reference/glossary/addons.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Add-ons - (Complementos)
+id: addons
+date: 2019-12-15
+full_link: /docs/concepts/cluster-administration/addons/
```

Comment:
Ideally, also add https://kubernetes.io/es/docs/concepts/cluster-administration/addons/ at the same time.

Reply:
I will take care of that now, thanks for the recommendation.